src/widgets/modals/FinaliseModal.js

```diff
@@ -63,7 +61,8 @@ export const FinaliseModal = props => {
   };
 
   return (
-    <ConfirmModal
+    <FinaliseModal
```

Comment:
This is the component `FinaliseModal` and you are returning `FinaliseModal`

This is the equivalent of `const someFunction = () => someFunction`

Reply:
@joshxg 
Should I just change component to NewConfirmModal?

Reply:
yes - but the props are slightly different between `NewConfirmModal` and `ConfirmModal`. Also you will be renaming `NewConfirmModal` to `ConfirmModal`

Reply:
@joshxg 
I will rename it as ConfirmForm as discussed